ci-operator/step-registry/ipi/install/vsphere/registry/ipi-install-vsphere-registry-ref.yaml

```diff
@@ -1,6 +1,6 @@
 ref:
   as: ipi-install-vsphere-registry
-  from: upi-installer
```

Comment:
Why not just use cli ?

Reply:
> Why not just use cli ?

Updated, but kinda wondering based on:
https://github.com/openshift/release/blob/master/ci-operator/config/openshift/installer/openshift-installer-release-4.5.yaml
will it even be the right version of `oc`